business/proxy_status.go

```diff
@@ -74,3 +75,50 @@ func xdsStatus(sent, acked string) string {
 	// Since the Nonce changes to uuid, so there is no more any time diff info
 	return "Stale"
 }
+
+func (in *ProxyStatus) GetConfigDump(namespace, pod string) (interface{}, error) {
```

Comment:
In general, the business package should try to return types from model package.
I guess this is returning a ConfigDump, I'd add a simple struct in the model package for this.
As it happens for other Istio objects (kubernetes vs model package) in the model package Kiali adds some processing (like adding additional permissions or validations info that it's used in next steps).

So, I guess same reasoning fits here, even if it's kinda duplicated of the kubernetes/envoy.go that you've added, but just to let business package continue returning objects from model package.

Reply:
Sure thing. I guess I've just missed in between all the work done in this PR. I guess that the handlers where the firsts methods I did to play around at the beginning.

I've added a wrapper type in the models layer for the ConfigDump and return it in here. To preserve the consistency across the layers.